chart/values.yaml

```diff
@@ -434,3 +434,34 @@ postgresql:
   enabled: true
   postgresqlPassword: postgres
   postgresqlUsername: postgres
+# Git sync
+gitSync:
+  enabled: true
+  persistence:
+    # Enable persistent volumes for storing synced dags
+    enabled: true
+    # Volume size for dags
+    size: 1Gi
+    # If using a custom storageClass, pass name here
+    storageClassName:
+  # git repo clone url
+  # ssh examples ssh://git@github.com/apache/airflow.git
+  # git@github.com:apache/airflow.git
+  # https example: https://github.com/apache/airflow.git
+  repo: https://github.com/apache/airflow.git
+  branch: v1-10-stable
```

Comment:
I will change it later to a variable. But for now iI think it's good. I plan to use the Helm chart not only for unit tests (which are great BTW) but also to run our Kubernetes tests (we are currently using some custom, simple templates for the kubernetes resources - replacing it with the helm chart from master and using images built locally will be a great way to do doggfooding.

Reply:
cool, that would be awesome 👍